### ISSUE TITLE
Web - OldDot - Secondary login is not validated

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#19916]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in to NewDot with any account
2. Log into OldDot with the same account and add a Secondary Login from Account Settings e.g. [mypersonal+validateTest@gmail.com](mailto:mypersonal+validateTest@gmail.com)
3. Open validate secondary log in email Verify secondary email for Expensify and copy the last part of the URL, i.e. /v/16/WSSWMNNM (Everything after the /v/)
4. In a text box - Formulate the URL by using the path obtained from step 3 http://staging.new.expensify.com/v/
- E.g http://staging.new.expensify.com/v/12116/WSSWMNNMH
5. Navigate to the compiled link

## Expected Result:
LHN should be displayed. Secondary login should be validated.

## Actual Result:
Secondary login is not validated

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.21.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/8bc2bb96-baaa-4274-b841-7528967a4a70

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sophiepintoraetz** [comment]
@kbecciv - can you please re-write the reproduction steps, they are hard to follow. Are you meant to validate the secondary login and then copy the URL? In your video, the email is unvalidated and yet you are saying the result should be validated.

**Comment by sophiepintoraetz** [comment]
I cannot reproduce this with the current steps:
1. I logged into new and old dot with sophie@expensifail.com
<img width="3005" alt="image" src="https://github.com/Expensify/App/assets/42940078/19699c54-9885-451d-b38b-b455f6a88fe9">


2. I added 11waltersrd@gmail.com as the secondary log in (I did not validate it)
3. I copied the link from the validation email and formatted it as suggested in the OP
<img width="1504" alt="image" src="https://github.com/Expensify/App/assets/42940078/d497f8ea-797b-4319-9a12-afaa9ff08118">

4. When I navigate to the compiled link I am presented with the magic code, which is what I'd expect given the account is unvalidated.
<img width="1504" alt="image" src="https://github.com/Expensify/App/assets/42940078/e9d6ddb8-975f-4634-b25c-8317b1e9ff82">

**Comment by sophiepintoraetz** [comment]
Closing until I get more context from @kbecciv

**Comment by sophiepintoraetz** [comment]
@kbecciv - I need you to provide feedback on my steps [here](https://github.com/Expensify/App/issues/19916#issuecomment-1573048124) - because I am not able to reproduce it.

**Comment by sophiepintoraetz** [comment]
Waiting until I have feedback.

**Comment by kbecciv** [comment]
Hello @sophiepintoraetz! I'm reopening the issue because the QA team is still able to reproduce the problem. I reviewed your testing [here](https://github.com/Expensify/App/issues/19916#issuecomment-1573048124), specifically on step 4. The secondary account should be validated after the user navigates to the compiled link and is presented with the magic code. I completely agree that the steps are hard to follow in TR, but if you can submit a TR update ticket and review the expected results here, that would be wonderful! Thank you and Merry Christmas

https://github.com/Expensify/App/assets/93399543/97104933-7199-472e-a755-9da37e3a8be2

**Comment by sophiepintoraetz** [comment]
Thanks @kbecciv for following up. I am going to close this issue for now - even with the clearer reproduction steps, this is:
1) Happening on OldDot, which we are working fast to deprecte (and therefore, not spending engineering effort on it)
2) There are no customer reports that I can see detailing this issue. 

If there is an increase of user reports (e.g. this is one that _may_  https://github.com/Expensify/Expensify/issues/346037 be associated) - then let's reopen it. until then, we are aware it is a bug but will not fix it.
